Installer.iss

```diff
@@ -2,7 +2,7 @@
 ; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
 
 #define MyAppName "EDDI"
-#define MyAppVersion "3.7.3-b1"
+#define MyAppVersion "3.7.3-a2"
```

Comment:
Did you want to commit this change to `develop`?

Reply:
Big Nope. I do not intend this line to merge into develop, rather that I will cherry-pick the winning number formats in the RESXs into CrowdIn, and the winning C~ code in here, and take it from there.

Reply:
Think of this branch as a public experiment.